runtests.py

```diff
@@ -1713,9 +1713,9 @@ def __init__(self, treefile, workdir, cleanup_workdir=True, stats=None,
         self.cleanup_workdir = cleanup_workdir
         self.stats = stats
         self.capture = capture
-        cython_syspath = [self.cython_root]
+        cython_syspath = [EndToEndTest.cython_root]
```

Comment:
Why this change?

Reply:
This is to clarify `cython_root` is class attribute.  I think it'd fit since this commit touches the attribute, but I can undo it if it creates blame noise.

Reply:
`self.xyz` is just fine for attributes defined in the class, but additionally works for subclasses and instances. I do not see a need to make this code less generic than it is.

Reply:
I understand, I'll revert to the previous code.  I made the change after unable to search for `self.cython_root` anywhere and thought a search to `EndToEndTest` would be faster to locate the variable's value.

Reply:
No need, I already updated your branch.